schemas/fragments/ReadStep.json

```diff
@@ -190,6 +307,15 @@
     },
     {
       "$ref": "#/$defs/Parquet"
+    },
+    {
+      "$ref": "#/$defs/Json"
```

Comment:
@zaychenko-sergei @EvilAlexei we'll need to schedule a UI task work to support new sources.